Source/WebDriver/socket/SessionHostSocket.cpp

```diff
@@ -212,4 +330,11 @@ void SessionHost::sendMessageToBackend(const String& message)
     sendWebInspectorEvent(sendEvent->toJSONString());
 }
 
+#if USE(INSPECTOR_SOCKET_SERVER)
+bool SessionHost::isRemoteBrowser() const
```

Comment:
This looks same with SessionHostGlib.cpp. Why don't you unify them?

Reply:
I'll unify them and move it to SessionHost.cpp.